APIs/schemas/flow_audio_coded.json

```
@@ -12,7 +12,7 @@
       ],
       "properties": {
         "media_type": {
-          "description": "Subclassification of the format using IANA assigned media types",
+          "description": "Subclassification of the format using IANA assigned media types, if available",
```

Should we try to write these in such a way so it doesn't seem that the field is optional if there isn't an IANA assigned media type?

```suggestion
          "description": "Subclassification of the format (prefer IANA assigned media types where available)",
```